tests/manage/z_cluster/nodes/test_automated_recovery_from_failed_nodes_proactive_IPI.py

```diff
@@ -6,6 +6,7 @@
     tier4b,
     ManageTest,
     aws_platform_required,
+    rosa_platform_required,
```

Comment:
I don't think that this will work because the tests will be marked as skipped by `aws_platform_required` marker and the marker `rosa_platform_required` won't undo it.